lib/issue.js

```diff
@@ -15,7 +15,12 @@ export async function issue({exchanger, exchange} = {}) {
     // run jsonata compiler; only `jsonata` template type is supported and this
     // was validated when the exchanger was created
     const credentials = await Promise.all(credentialTemplates.map(
-      ({template: t}) => jsonata(t).evaluate(variables)));
+      async ({template: t}) => {
+        const {vc, ...rest} = variables;
+        const parsedVc = vc ? JSON.parse(
+          JSON.stringify(await jsonata(vc).evaluate(rest))) : null;
+        return jsonata(t).evaluate(parsedVc ? {vc: parsedVc} : variables);
```

Comment:
We're specifically trying to avoid doing something like this -- which special cases the `vc` variable. It should work without having to do this. A demonstration of what we want to work is here in the jsonata playground: https://try.jsonata.org/TVb5W8K3Q

Reply:
@dlongley Updated [4273e76](https://github.com/digitalbazaar/bedrock-vc-delivery/pull/30/commits/4273e76e2ba9c2a922f09a943519bb8f94860986)